### PR TITLE
docs(developer): kmc-generate

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

Relates-to: #12641

### CODE REVIEW

File: developer/docs/help/reference/kmc/cli/reference.md [hunk 1]
```diff
@@ -294,6 +306,163 @@ For more information on the purpose of `analyze osk-char-use` and
...
+`-o, --out-path <path>`
+
+: Specifies the parent folder for the project; the folder may already exist. The
+  project will be generated in a new folder named with the ID of the project
+  under this path, and that project folder must not exist.
+
+`-n, --name <name>`
+
+: Keyboard descriptive name, referenced in the keyboard `<info name` attribute
+  (default: the ID of the project)
```
Comment (darcywong00) [inline developer/docs/help/reference/kmc/cli/reference.md:392]: Maybe suggest: "Do not include `"keyboard"` in the name"

File: developer/docs/help/reference/kmc/cli/reference.md [hunk 2]
```diff
@@ -294,6 +306,163 @@ For more information on the purpose of `analyze osk-char-use` and
...
+  tag may be specified, with each tag prefixed with `-L` (default: no languages).
+  The tags are referenced in the package metadata.
+
+`-a, --author <author-name>`
+
+: The name of model author, referenced in package metadata (default: blank)
+
+`-d, --description <description>`
+
+: A short description of the project, in Markdown. (default: keyboard name)
```
Comment (darcywong00) [inline developer/docs/help/reference/kmc/cli/reference.md:464]: lexical model name?

File: developer/docs/help/reference/kmc/cli/reference.md [hunk 3]
```diff
@@ -294,6 +306,163 @@ For more information on the purpose of `analyze osk-char-use` and
...
+: [`&copyright`](/developer/language/reference/copyright) holder for the
+  project. Do not include the '(C)' or '&copy;' prefixes (default: the author
+  of the keyboard)
+
+`-v, --version <version-string>`
+
+: [`&version`](/developer/language/reference/version) of the keyboard, (default:
+  "1.0").
+
+`-L, --language-tag <bcp-47 tag>`
```
Comment (darcywong00) [inline developer/docs/help/reference/kmc/cli/reference.md:348]: I just noticed.

Is lower-case `-l` already in use?

Comment (mcdurdin) [reply developer/docs/help/reference/kmc/cli/reference.md:348]: Yes, `-l, --log-level` is a global parameter

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.140-alpha](https://keyman.com/downloads/releases/alpha/18.0.140)

**Pull request merged**